### DISCUSSION TITLE
Error Summary Tab does not show up anything

### DISCUSSION THREAD

**Discussion by VikasKrishnamraju** [ludeknovy/jtl-reporter#230] [General]
Hi i see that even though there are errors in the jtl file nothing shows up in the error summary tab. is this tab still under development or is there something im missing here. Below is an example

<img width="1717" alt="Screenshot 2023-09-01 at 2 47 35 PM" src="https://github.com/ludeknovy/jtl-reporter/assets/24792711/62b935f5-f167-468d-b1cf-b52d2421ffc8">

**Comment by ludeknovy** [comment]
Hi @VikasKrishnamraju 
It looks like a bug. Could you replace all sensitive data in your jtl/csv file and share it? No need for the whole file, just a sample with the error lines so I could investigate it?

**Reply by ludeknovy** [reply]
and do you use v4.7.6 and higher of backend service?

**Reply by VikasKrishnamraju** [reply]
Hi @ludeknovy thank you for you quick response. Let me check on the version and get back to you.
And is there an easy way to share the jtl file by replacing sensitive data as it a 60 min duration execution

**Reply by ludeknovy** [reply]
Just share a subset of the data, and in your favorite editor it should be easy to replace eg the hostname with dummy data using the find and replace all functionality.

**Reply by VikasKrishnamraju** [reply]
Does the below data help.. sample subset

timeStamp,elapsed,label,responseCode,responseMessage,threadName,dataType,success,failureMessage,bytes,sentBytes,grpThreads,allThreads,URL,Latency,IdleTime,Connect

1693471449906,16376,/api/authenticationservice/Authentication/authenticationstatus,500,Internal Server Error,Thread Group 1-2,,false,,302,4819,10,10,https://sample-env.com/api/authenticationservice/Authentication/authenticationstatus,16376,0,0

1693471424197,42087,/api/authenticationservice/Authentication/authenticationstatus,500,Internal Server Error,Thread Group 1-3,,false,,302,4819,10,10,https://sample-env.com/api/authenticationservice/Authentication/authenticationstatus,42087,0,0

1693471502638,22,/api/authenticationservice/Authentication/authenticationstatus,500,Internal Server Error,Thread Group 1-10,,false,,302,4819,10,10,https://sample-env.com/api/authenticationservice/Authentication/authenticationstatus,22,0,0

1693471466557,3,/api/authenticationservice/Authentication/tokencookie,403,Forbidden,Thread Group 1-5,,false,,290,1148,10,10,https://sample-env.com/api/authenticationservice/Authentication/tokencookie,3,0,0

1693471466558,5,/api/authenticationservice/Authentication/tokencookie,403,Forbidden,Thread Group 1-8,,false,,290,1148,10,10,https://sample-env.com/api/authenticationservice/Authentication/tokencookie,5,0,0

1693471466657,4,/api/authenticationservice/Authentication/tokencookie,403,Forbidden,Thread Group 1-6,,false,,290,1148,10,10,https://sample-env.com/api/authenticationservice/Authentication/tokencookie,4,0,0

**Reply by ludeknovy** [reply]
Thanks! could you please also provide the headers (column names) ?

**Reply by VikasKrishnamraju** [reply]
opps my bad !! missed it. i have updated the above comment

the version im using are  fe : 4.8.7 & be : 4.8.4

**Reply by ludeknovy** [reply]
ahh, ok, be and fe are out of sync then. The error summary feature was released in https://github.com/ludeknovy/jtl-reporter-be/releases/tag/v4.8.6

so it cannot really work for you.

**Reply by VikasKrishnamraju** [reply]
Got it now !! so the we need fe : 4.8.7 and be : 4.8.8

and we need this below migration piece as well right
migration:
    image: novyl/jtl-reporter-be:v4.8.8

Will update to latest and check.

Thank you @ludeknovy  for your quick reponses and input.. much appreciated

**Reply by ludeknovy** [reply]
Yes, you need to always keep the migration in sync with be service

**Reply by ludeknovy** [reply]
The rule of thumb is always to use the same version as provided in the docker-compose file https://github.com/ludeknovy/jtl-reporter/blob/main/docker-compose.yml

**Reply by VikasKrishnamraju** [reply]
Cool !! can i know what is the use case of the migration service
